pages/About.js

```diff
@@ -0,0 +1,16 @@
+import NavBar from "../components/NavBar"
+import AboutInfo from "../components/AboutInfo"
+import globalStyles from "../styles/global";
+
+
+export default function About() {
```

Comment:
What do you think about renaming the file to `about.js`? It makes the URL look for standard. That's also how Next.js does it in their docs. https://nextjs.org/docs/basic-features/pages

Reply:
renamed after looking at their docs.